3.5.x apoc.schema.nodes fails

Neo.ClientError.Procedure.ProcedureCallFailed: Failed to invoke procedure `apoc.schema.nodes`: Caused by: java.lang.IncompatibleClassChangeError: Found interface org.neo4j.storageengine.api.schema.PopulationProgress, but class was expected

--- a/build.gradle
+++ b/build.gradle
@@ -38,7 +38,7 @@ description = """neo4j-apoc-procedures"""
 
 ext {
     // NB: due to version.json generation by parsing this file, the next line must not have any if/then/else logic
-    neo4jVersion = "3.5.3"
+    neo4jVersion = "3.5.7"
     // instead we apply the override logic here
     neo4jVersionEffective = project.hasProperty("neo4jVersionOverride") ? project.getProperty("neo4jVersionOverride") : neo4jVersion
     testContainersVersion = '1.11.0'

